lib/merge-yaml

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/python3
+
+import json
+import hiyapyco
```

Comment:
This is not installed by default, but even worse - there is no Fedora package for it.
If standard yaml library doesn't support merging repeating keys, then maybe better load individual yaml parts separately and merge them in python (the thing that currently is done with cat+sed)? Merging dicts apparently is not so trivial (https://stackoverflow.com/q/7204805), but doable.
Then, you can trivially output de-duplicated yaml, as well as json.